src/Package/Impl/Publishing/Handlers/RmdPublishHandler.cs

```diff
@@ -28,7 +29,7 @@ internal sealed class RmdPublishHandler : IMarkdownFlavorPublishHandler {
             string outputFile = Path.GetFileName(outputFilePath);
             string outputFolder = Path.GetDirectoryName(outputFilePath).Replace('\\', '/');
             // Run rmarkdown::render
-            return $"\"rmarkdown::render(\'{inputFile}\', output_format=\'{format}\', output_file='{outputFile}',  output_dir='{outputFolder}')\"";
```

Comment:
But if we save it using native encoding (when it can be thus saved), won't the original reported issue re-manifest?

Also, note that OS locale and R locale do not necessarily match (which was a root cause of many of our bugs regarding Unicode).